tests/checks/test-VIPRestrictedCommandsCheck.php

```diff
@@ -148,7 +148,9 @@ public function testOther() {
 			array( 'slug' => 'wp_is_mobile',              'level' => 'Warning', 'description' => "wp_is_mobile() is not batcache-friendly, please use <a href=\"http://vip.wordpress.com/documentation/mobile-theme/#targeting-mobile-visitors\">jetpack_is_mobile()</a>", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),
 			array( 'slug' => 'show_admin_bar',            'level' => 'Blocker', 'description' => "The WordPress.com admin bar cannot be removed as it’s integral to the user experience on WordPress.com", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),
 			array( 'slug' => 'serialize',                 'level' => 'Blocker', 'description' => "Serialized data has <a href='https://www.owasp.org/index.php/PHP_Object_Injection'>known vulnerability problems</a> with Object Injection. JSON is generally a better approach for serializing data.", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),
-			array( 'slug' => 'unserialize',               'level' => 'Blocker', 'description' => "Serialized data has <a href='https://www.owasp.org/index.php/PHP_Object_Injection'>known vulnerability problems</a> with Object Injection. JSON is generally a better approach for serializing data.", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),		);
+			array( 'slug' => 'unserialize',               'level' => 'Blocker', 'description' => "Serialized data has <a href='https://www.owasp.org/index.php/PHP_Object_Injection'>known vulnerability problems</a> with Object Injection. JSON is generally a better approach for serializing data.", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),
+			array( 'slug' => 'get_intermediate_image_sizes',	'level' => 'Blocker', 'description' => "Intermediate images do not exist on VIP.", 'file' => 'VIPRestrictedCommandsCheck8.inc', 'lines' => ++$line ),
```

Comment:
Would be nice to provide some more detailed explanation. Here's a snippet explaining the empty array returned from `get_intermediate_image_sizes` function to the client in one of the support tickets

> Image sizes at WordPress.com VIP platform are passed as an argument (?w=xxx). Sizes are filtered out mainly to prevent the thumbnail creation that WordPress core does since thumbnails are being generated on the fly at the platform.